.idea/.generators

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Thanks for adding the files to the `.gitignore`.  Could you remove the rest of the .idea files?